toolz/tests/test_dicttoolz.py

```diff
@@ -1,17 +1,41 @@
-from toolz.utils import raises
+import sys
+import collections
+
 from toolz.dicttoolz import (merge, merge_with, valmap, keymap, update_in,
                              assoc, dissoc, keyfilter, valfilter, itemmap,
                              itemfilter)
 
+# Python 2.6 compatibility
+_OrderedDict = getattr(collections, 'OrderedDict',
+                       type('OrderedDict', (dict,), {}))
+PY26 = not all(i <= j for i, j in zip((2, 7), sys.version_info[:2]))
```

Comment:
I wasn't sure at first, so I went for the option that I was sure worked. But you're right, tuples are compared lexicographically apparently, so `sys.version_info < (2, 7)` should work. I don't think `sys.version_info <= (2, 6)` does, but `sys.version_info[:2] <= (2, 6)` would.